.gitignore

```diff
@@ -1,10 +1,91 @@
 
-# Maven
```

Comment:
It would be great if you could leave the file to a minimum of what is required. If you have something in addition, please create an additional PR with the changes you think are helpful.